fastdtw/__init__.py

```diff
@@ -0,0 +1,6 @@
+try:
+    from ._fastdtw import fastdtw
```

Comment:
IMO, If it is not a problem, it could be better to expose it. Some guys may prefer non approximated results.